editor/js/outcomes.js

```diff
@@ -268,6 +313,12 @@ function unloadAnalysis(analysisNode, analyses) {
   }
 }
 
+/**
+ * Fetches a processed_data record defined in the form and adds it to
+ * `processes`.
+ * @param {!Node} node Root node for the Data record.
+ * @param {!jspb.Map<string, !proto.ord.Data>} processes
```

Comment:
I don't love the variable name `processes` here (and previous function names like `loadProcess`). Is it worth changing this to processedData, loadProcessedData, etc.?

Reply:
I'll do this as part of #349 so we only change JSDoc in this PR; sound good?

Reply:
Yep, sounds good